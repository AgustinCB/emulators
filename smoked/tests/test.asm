.data
INTEGER 1
.code
CONSTANT 0
