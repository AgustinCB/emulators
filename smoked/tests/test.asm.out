0 - SimpleValue(Integer(1))
